app/bin/service/analyzer.dart

```diff
@@ -5,10 +5,26 @@
 import 'dart:io';
 
 import 'package:appengine/appengine.dart';
+import 'package:logging/logging.dart';
+
+final Logger logger = new Logger('pub.analyzer');
 
 void main() {
   useLoggingPackageAdaptor();
 
+  final String flutterSdkDir = Platform.environment['FLUTTER_SDK'];
+  if (flutterSdkDir == null) {
+    logger.warning('FLUTTER_SDK is not set, assuming flutter is in PATH.');
```

Comment:
These logging messages are not going where they are supposed to go without having setup the appengine services first.